addons/hr/static/src/scss/hr.scss

```
@@ -4,7 +4,38 @@
 }
 
 .o_employee_form {
+    .o_employee_avatar {
+        position: absolute;
+        top: 60px;
+        right: 10px;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        .o_employee_availability {
+            position: absolute;
+            top: -5px;
+            right: -5px;
+            padding-bottom: 1px;
+            border-radius: 50%;
+            background-color: white;
```

So it looks nicer on dark mode :-D

```suggestion
            background-color: $o-view-background-color;
```